src/content/developers/docs/scaling/state-channels/index.md

```diff
@@ -165,6 +165,8 @@ Channel-based payments have the following advantages:
 
 3. **Latency**: Off-chain transactions conducted between channel participants can be settled instantly, if both parties cooperate, reducing delays. In contrast, sending a transaction on Mainnet requires waiting for nodes to process the transaction, produce a new block with the transaction, and reach consensus. Users may also need to wait for more block confirmations before considering a transaction finalized.
 
+4. **Cost**: State channels are particularly useful in situations where a set of participants will be exchanging many state updates over a long period of time. This is because the only cost incurred by the channel is the initial setup and closure of the state channel smart contract. Consequently, every additional intermediary state change in the channel will be cheaper than the last as the settlement cost is distributed accordingly.
```

Comment:
```suggestion
4. **Cost**: State channels are particularly useful in situations where a set of participants will exchange many state updates over a long period. This is because the only cost incurred by the channel is the initial setup and closure of the state channel smart contract. Consequently, every additional intermediary state change in the channel will be cheaper than the last as the settlement cost is distributed accordingly.
```